.gitignore

```diff
@@ -1,4 +1,3 @@
-*.mode*
```

Comment:
This change seems unrelated.

Reply:
This seems needed to make git happy:

>* .gitignore:
>Remove a confusingly widely scoped gitignore. It doesn't seem to be a file
extension that Xcode emits anymore, and conflicts with the sandbox profiles
below.